packages/slice-machine/pages/onboarding.tsx

```diff
@@ -93,7 +92,10 @@ const PushPagesSlide = () => (
 const OnboardingGrid = ({
   children,
 }: {
-  children: ReactElement | ReadonlyArray<ReactElement>;
+  children:
```

Comment:
If you want to type functional component, you can use React.FunctionComponent<Props> this type includes children as props :)